### PR TITLE
DOCS: Remove examples

### PR DESCRIPTION
**Pull request by Samuelopez-ansys**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -117,6 +90,7 @@ doc-no-examples = [
     "scikit-rf>=0.30.0,<1.4",
     "pyvista[io]>=0.38.0,<0.45",
```
Comment (SMoraisAnsys) [inline pyproject.toml:91]: Not sure but can't we also get rid of matplotlib, scikit-rf and pyvista dependencies for doc build?

**Review by SMoraisAnsys — COMMENTED** [review]
Two minor comments:
- can you update the doc dependencies to remove the one that are not used (pyvista, ...)?
- can you update the Makefile in the doc folder ?

File: pyproject.toml (hunk 1 above)
Comment (Samuelopez-ansys) [reply pyproject.toml]: @SMoraisAnsys Done

**Review by SMoraisAnsys — APPROVED** [review]
LGTM

**Review by MaxJPRey — APPROVED** [review]

**Pull request merged**